cg/cli/workflow/mutant/base.py

```diff
@@ -99,3 +99,39 @@ def start_available(context: click.Context, dry_run: bool = False):
             exit_code = EXIT_FAIL
     if exit_code:
         raise click.Abort
+
+
+@mutant.command("store-available")
+@OPTION_DRY
+@click.pass_context
+def store_available(context: click.Context, dry_run: bool) -> None:
+    """Run QC checks and store bundles for all finished analyses in Housekeeper."""
+
+    analysis_api: MutantAnalysisAPI = context.obj.meta_apis["analysis_api"]
+
+    exit_code: int = EXIT_SUCCESS
+
+    analysis_api.run_qc_and_fail_analyses(dry_run=dry_run)
+
+    for case_obj in analysis_api.get_cases_to_store():
+        LOG.info("Storing deliverables for %s", case_obj.internal_id)
+        try:
+            context.invoke(store, case_id=case_obj.internal_id, dry_run=dry_run)
```

Comment:
As a design pattern we want to move away from calling other CLI commands. But that is out of scope for your PR to fix - leave this as it is! 👍

Reply:
Will keep it in mind for the future!